massa-bootstrap/src/lib.rs

```diff
@@ -349,12 +385,19 @@ impl BootstrapServer {
                     // load cache if absent
                     if bootstrap_data.is_none() {
                         massa_trace!("bootstrap.lib.run.select.accept.cache_load.start", {});
+
+                        // note that all requests are done simultaneously
+                        // except for the consensus graph that is done after the others
+                        // this is done to ensure that the execution bootstrap state is older than the consensus state
+                        // (and not more recent which could cause problems with execution bootstrap)
```

Comment:
Could you write maybe in a read.md what could exactly happen ? I see it's not ok but maybe with a few graphs :)

Reply:
I improved the comment